data/devices/samsung/00_galaxy_s.yml

```diff
@@ -729,3 +729,192 @@
+      - /dev/block/bootdevice/RECOVERY
+      - /dev/block/platform/11120000.ufs/by-name/RECOVERY
+      - /dev/block/sda8
+    extra:
+      - /dev/block/bootdevice/RADIO
+      - /dev/block/platform/11120000.ufs/by-name/RADIO
+      - /dev/block/sda10
+
+  boot_ui:
+    supported: true
+    flags:
+      - TW_HAS_DOWNLOAD_MODE
+    graphics_backends:
+      - fbdev
+    brightness_path: /sys/devices/platform/panel_drv@001/backlight/panel/brightness
+    max_brightness: 36600
+    default_brightness: 16200
+    pixel_format: ABGR_8888
+    theme: portrait_hdpi
+
+- name: Samsung Galaxy S8+ (Exynos)
+  id: dream2lte
+  codenames:
+    # Regular variant
+    - SM-G955F
+    - dream2lte
+    - dream2ltexx
+  architecture: arm64-v8a
+
+  block_devs:
+    base_dirs:
+      - /dev/block/bootdevice/by-name
+      - /dev/block/platform/11120000.ufs/by-name
+    system:
+      - /dev/block/bootdevice/by-name/SYSTEM
+      - /dev/block/platform/11120000.ufs/by-name/SYSTEM
+      - /dev/block/sda17
+    cache:
+      - /dev/block/bootdevice/by-name/CACHE
+      - /dev/block/platform/11120000.ufs/by-name/CACHE
+      - /dev/block/sda18
+    data:
+      - /dev/block/bootdevice/by-name/USERDATA
+      - /dev/block/platform/11120000.ufs/by-name/USERDATA
+      - /dev/block/sda24
+    boot:
+      - /dev/block/bootdevice/by-name/BOOT
+      - /dev/block/platform/11120000.ufs/by-name/BOOT
+      - /dev/block/sda7
+    recovery:
+      - /dev/block/bootdevice/RECOVERY
+      - /dev/block/platform/11120000.ufs/by-name/RECOVERY
+      - /dev/block/sda8
+    extra:
+      - /dev/block/bootdevice/RADIO
+      - /dev/block/platform/11120000.ufs/by-name/RADIO
+      - /dev/block/sda10
+
+  boot_ui:
+    supported: true
+    flags:
+      - TW_HAS_DOWNLOAD_MODE
+    graphics_backends:
+      - fbdev
+    brightness_path: /sys/devices/platform/panel_drv@001/backlight/panel/brightness
+    max_brightness: 36600
+    default_brightness: 16200
+    pixel_format: ABGR_8888
+    theme: portrait_hdpi
+
+- name: Samsung Galaxy S8 (Snapdragon)
+  id: dreamqlte
+  codenames:
+    # Regular variant
+    - SM-G950U
+    - dreamqlte
+    - lineage_dreamqlte
+  architecture: arm64-v8a
+
+  block_devs:
+    base_dirs:
+      - /dev/block/bootdevice/by-name
+    system:
+      - /dev/block/bootdevice/by-name/system
+    cache:
+      - /dev/block/bootdevice/by-name/cache
+    data:
+      - /dev/block/bootdevice/by-name/userdata
+    boot:
+      - /dev/block/bootdevice/by-name/boot
+    recovery:
+      - /dev/block/bootdevice/recovery
+    extra:
+      - /dev/block/bootdevice/modem
+
+  boot_ui:
+    supported: true
+    flags:
+      - TW_HAS_DOWNLOAD_MODE
+    graphics_backends:
+      - fbdev
+    brightness_path: /sys/class/leds/lcd-backlight/brightness
+    max_brightness: 36500
+    default_brightness: 15000
+    pixel_format: RGBX_8888
+    theme: portrait_hdpi
+
+- name: Samsung Galaxy S8+ (Snapdragon)
+  id: dream2qlte
+  codenames:
+    # Regular variant
+    - SM-G955U
```

Comment:
The SM-G955U is the US carrier unlocked version, but that version doesn't have an unlocked bootloader.

Can we limit this to just `dream2qltezh` and `dream2qltezhn`? As far as I know, those are the only Snapdragon S8/S8+ devices that have an unlocked bootloader.